src/GeositeFramework/plugins/launchpad/style.css

```diff
@@ -26,3 +26,41 @@
     list-style-type: none;
     padding: 0;
 }
+
+.launchpad-plugin .plugins .button.truncate-toggle, .scenarios .button.truncate-toggle {
+    padding: 2px !important;
+    font-size: 0.7em !important;
```

Comment:
Consider removing `!important` from these lines. There didn't seem to be a visual difference when I removed them during testing.